myosuite/envs/myo/myochallenge/bimanual_v0.py

```diff
@@ -258,6 +258,8 @@ def get_reward_dict(self, obs_dict):
         goal_dis = np.array(
             [[np.abs(np.linalg.norm(obj_pos - goal_pos, axis=-1))]])
         
+        touching_vec = obs_dict["touching_body"][0][0] if obs_dict['touching_body'].ndim == 3 else obs_dict['touching_body']
```

Comment:
wha this vector about?

Reply:
This gives information on what the object is touching (out of start/goal/myohand/MPL/env). This is used later is the 'solved' to see if the object is touching the pillow.

Reply:
would it be enought to touch once or it needs to be placed on the pillar i.e. stay there for some time?

Reply:
[3c19523](https://github.com/MyoHub/myosuite/pull/231/commits/3c19523e9ae6298868a58f087cb58d7f2dd86df3) Updated here